src/legacy/ui/public/field_editor/components/field_format_editor/editors/url/url.js

```diff
@@ -113,6 +120,44 @@ export class UrlFormatEditor extends DefaultFormatEditor {
     });
   }
 
+  renderWidthHeightParameters = () => {
+    const { error } = this.state;
+    const width = this.sanitizeNumericValue(this.props.formatParams.width);
+    const height = this.sanitizeNumericValue(this.props.formatParams.height);
+    return (
+      <Fragment>
+        <EuiFormRow
+          label={<FormattedMessage id="common.ui.fieldEditor.url.widthLabel" defaultMessage="Width" />}
```

Comment:
:thinking: Just wondering if we should set this to "Maximum Width", since it would reflect better what it currently does, or if the "Width" is actually better, because most people don't mind the difference and it's easier to understand. @nreese since you pushed the original issue, what would you prefer?

Reply:
I think width is find since it directly corresponds the `img` width attribute